content/patterns/alert/examples/alert.html

```diff
@@ -20,85 +20,84 @@
+          <!--  以下のscript要素には、アラート要素に挿入される内容が含まれています。 -->
           <script type="text/template" id="alert-template">
-            <p>Hello</p>
+            <p>こんにちは</p>
           </script>
         </div>
-        <div role="separator" id="ex_end_sep" aria-labelledby="ex_end_sep ex_label" aria-label="End of"></div>
+        <div role="separator" id="ex_end_sep" aria-labelledby="ex_label ex_end_sep" aria-label="の終了"></div>
       </section>
 
       <section>
-        <h2>Accessibility Features</h2>
+        <h2>アクセシビリティの特徴</h2>
         <p>
-          Because an alert is for critical information, assistive technologies may provide special behaviors designed to help call attention to changes in the text of an alert.
-          For example, screen readers may interrupt all other speech and preface announcement of the new alert text with a special sound or phrase.
+          アラートは重要な情報のため、支援技術はアラートのテキストの変更に注意を向けるための特別な動作を提供する場合があります。例えば、スクリーンリーダーは他のすべての読み上げを中断し、新しいアラートテキストの前に特別な音又はフレーズでアナウンスする場合があります。
         </p>
       </section>
 
       <section>
-        <h2 id="kbd_label">Keyboard Support</h2>
-        <p>No keyboard interaction needed.</p>
+        <h2 id="kbd_label">キーボードのサポート</h2>
+        <p>キーボードによるインタラクションは必要ありません。</p>
       </section>
 
       <section>
-        <h2 id="rps_label">Role, Property, State, and Tabindex Attributes</h2>
+        <h2 id="rps_label">ロール、プロパティ、ステート、及び tabindex 属性</h2>
         <table aria-labelledby="rps_label" class="data attributes">
           <thead>
             <tr>
-              <th scope="col">Role</th>
-              <th scope="col">Attribute</th>
-              <th scope="col">Element</th>
-              <th scope="col">Usage</th>
+              <th scope="col">ロール</th>
+              <th scope="col">属性</th>
+              <th scope="col">要素</th>
+              <th scope="col">使用法</th>
             </tr>
           </thead>
           <tbody>
             <tr data-test-id="alert-role">
               <th scope="row"><code>alert</code></th>
               <td></td>
               <td><code>div</code></td>
-              <td>Identifies the element as the container where alert content will be added or updated.</td>
+              <td>アラートのコンテンツが追加又は更新されるコンテナとしての要素を識別します。</td>
             </tr>
             <tr data-test-id="test-not-required">
               <td></td>
               <th scope="row">
                 <code>aria-live=<q>assertive</q></code>
               </th>
-              <td>Implicit on <code>div</code></td>
+              <td><code>div</code> に暗黙的</td>
               <td>
                 <ul>
-                  <li>This does not have to be declared in the code because it is implicit in the alert role.</li>
-                  <li>Tells assistive technologies to interrupt other processes to provide users with immediate notification of relevant alert container changes.</li>
+                  <li>これは alert のロールにおいて暗黙的であるため、コードで宣言する必要はありません。</li>
```

Comment:
alert のロール→alert ロール
「暗黙的である」は慣れている人には意味がわかると思うのですが、少し補った方が読みやすいかもしれません。「暗黙的に割り当てられる」くらい?

```suggestion
                  <li>これは alert ロールによって暗黙的に割り当てられるため、コードで宣言する必要はありません。</li>
```